src/cogs/interactions/slash/mod.py

```diff
@@ -12,21 +12,11 @@
 class SlashMod(commands.Cog):
     def __init__(self, client: DiscordBot):
         self.client = client
-        self.tasks = [self.check_mutes_loop]
-        for task in self.tasks:
-            if not task.is_running():
-                task.start()
-
-    async def cog_unload(self):
-        for task in self.tasks:
-            task.cancel()
 
     @commands.Cog.listener()
     async def on_ready(self):
-        await self.client.wait_until_ready()
-        for task in self.tasks:
-            if not task.is_running():
-                task.start()
+        if not self.check_mutes_loop.is_running():
+            self.check_mutes_loop.start()

```

Comment:
Pull the latest changes from `main` before you create PRs. This isn't the updated version